op-node/flags/flags.go

```diff
@@ -162,5 +159,12 @@ var optionalFlags = append([]cli.Flag{
 	SnapshotLog,
 }, p2pFlags...)
 
-// Flags contains the list of configuration options available to the binary.
+// GlobalFlags contains the list of configuration options available to every command.
+var GlobalFlags = []cli.Flag{
+	LogLevelFlag,
+	LogFormatFlag,
+	LogColorFlag,
+}
+
+// Flags contains the list of configuration options available to the start command.
```

Comment:
We don't use a logger in the p2p commands. Maybe the other commands later, so it's fine to change, but still surprised.